### ISSUE TITLE
No discernable text on social media icons

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-theme-2015#204]
I have been playing around with the screen reader i downloaded. As I tab over the social media icons, the screen reader just states "link" "link" "link" etc...

There are multiple scanners I use to test pages, and here is the suggested fix:

To solve this violation, you need to:
Fix at least one (1) of these issues:
Element does not have text that is visible to screen readers
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty or not visible
Element's default semantics were not overridden with role="presentation"
Element's default semantics were not overridden with role="none"
And
Fix the following:
Element is in tab order and does not have accessible text

Not sure if you can do anything with that :) Thanks!

**Comment by joelworsham** [comment]
So I looked and they did already have an implementation of screen reader text, but perhaps they weren't being picked up by those scanners.

Anyways, I added an `aria-label` to them. Let's see if that takes care of it!
